contracts/core/State.sol

```diff
@@ -21,7 +22,8 @@ import "../interfaces/IWrbtcERC20.sol";
  * */
 contract State is Objects, ReentrancyGuard, Ownable {
 	using SafeMath for uint256;
-	using EnumerableBytes32Set for EnumerableBytes32Set.Bytes32Set;
+	using EnumerableAddressSet for EnumerableAddressSet.AddressSet; // enumerable map of addresses
```

Comment:
```suggestion
	using EnumerableAddressSet for EnumerableAddressSet.AddressSet; // enumerable map of addresses
```
```suggestion
```
can be removed as not used and EnumerableBytes32Set has methods for addresses adding/removing

Reply:
We still use the EnumerableAddressSet for referralsList storage